service/Dockerfile

```diff
@@ -1,17 +1,16 @@
-FROM openjdk:8-jdk-alpine
+FROM openjdk:11-jdk
 
 ARG COMMIT_SHA=<not-specified>
 ARG BUILD_FILE_NAME=application.jar
 
-WORKDIR /home/java/app
+WORKDIR /home/java
 
-RUN adduser -D -S -s /sbin/nologin -G root java && \
-    echo "mia_template_service_name_placeholder: $COMMIT_SHA" >> ./commit.sha
+RUN adduser --disabled-password --shell /sbin/nologin --ingroup root java && \
+    echo "custom-plugin: $COMMIT_SHA" >> ./commit.sha
```

Comment:
we used `mia_template_service_name_placeholder` because this string gets interpolated with the service name defined by the user upon service repository creation from the Console. I'd keep it.